src/lib/mina_metrics/mina_metrics.ml

```diff
@@ -47,7 +47,7 @@ module TextFormat_0_0_4 = struct
     match Float.classify v with
     | Normal | Subnormal | Zero ->
         Fmt.float f v
-    | Infinite when v > 0.0 ->
+    | Infinite when Float.is_positive v ->
```

Comment:
This seems to have the same off-by-one semantic problem the other line I pointed at did.